pkg/simulation/simulation.go

```diff
@@ -36,6 +48,17 @@ func NewSimEnv(parameters *config.ConfigParameters, healthInfo *health.HealthInf
 		AgentsPerFloor: parameters.AgentsPerFloor,
 		logger:         *log.WithFields(log.Fields{"reporter": "simulation"}),
 		stateLog:       logging.NewLogState(parameters.LogFileName),
+		agentNewFuncs: map[agent.AgentType]AgentNewFunc{
+			agent.Team1Agent1: agent1.New,
```

Comment:
This is not really your problem, but can we rename team1's agent1 and agent2 packages to team1Agent1 and team1Agent2 so it is consistent with the naming of all other agents ? Or should I just create an issue for them so we don't touch code we don't own and tell them to do it themselves?

Reply:
Ah, I was too slow, I'll just make an issue

Reply:
I think things like this should be done in different commits, just to get things moving faster.

Reply:
Also just realised that these are the test agents, so it is not really a problem to have a different naming convention. Also I didn't even realise I am in Team1 lol.